lib/widgets/fields/alignment_field.dart

```diff
@@ -0,0 +1,155 @@
+                ],
+              ),
+              Row(
+                children: [
+                  customRadioButton(4),
+                  customRadioButton(5),
+                  customRadioButton(6),
+                ],
+              ),
+              Row(
+                children: [
+                  customRadioButton(7),
+                  customRadioButton(8),
+                  customRadioButton(9),
+                ],
+              ),
+            ],
+          ),
+        ),
+      ],
+    );
+  }
+
+  Widget customRadioButton(int index) {
+    BorderRadius? borderRadius;
+    String text;
+    Alignment alignment;
+
+    switch (index) {
+      case 1:
+        borderRadius = const BorderRadius.only(topLeft: Radius.circular(7));
+        text ="Top Left";
+        alignment = Alignment.topLeft;
+        break;
+      case 2:
+        text ="Top";
+        alignment = Alignment.topCenter;
+        break;
+      case 3:
+        borderRadius = const BorderRadius.only(topRight: Radius.circular(7));
+        text ="Top Right";
+        alignment = Alignment.topRight;
+        break;
+      case 4:
+        text ="Left";
+        alignment = Alignment.centerLeft;
+        break;
+      case 5:
+        text ="Center";
+        alignment = Alignment.center;
+        break;
+      case 6:
+        text ="Right";
+        alignment = Alignment.centerRight;
+        break;
+      case 7:
+        borderRadius = const BorderRadius.only(bottomLeft: Radius.circular(7));
+        text ="Bottom Left";
+        alignment = Alignment.bottomLeft;
+        break;
+      case 8:
+        text ="Bottom";
+        alignment = Alignment.bottomCenter;
+        break;
+      case 9:
+        borderRadius = const BorderRadius.only(bottomRight: Radius.circular(7));
+        text ="Bottom Right";
+        alignment = Alignment.bottomRight;
+        break;
+      default:
+        borderRadius = BorderRadius.circular(0);
+        alignment = Alignment.center;
+        text ="";
+        break;
+    }
+
+    return InkWell(
+      onTap: () {
+        setState(() {
+          selectedOption = index;
+          widget.onChanged(alignment);
+        });
+      },
+        onHover: (val){
+          setState(() {
+            isHover=val;
+            onHoverOption=index;
+          });
+        },
+      child: Container(
+          padding: const EdgeInsets.only(bottom: 2),
+            margin: const EdgeInsets.all(0),
+            alignment: Alignment.center,
+            // height: 23,
+            width: 105.3,
+            decoration: BoxDecoration(
+              color: const Color(0xFF36343B),
+              border: Border.all(
+                color: (selectedOption == index)? Colors.white:
+                const Color(0xFF49454F),
+                width: 1,
+              ),
+              borderRadius: borderRadius,
+            ),
+            child: Center(
+              child: (selectedOption == index)? Text(text):
```

Comment:
Rather than passing widget on Condition Checks , pass value to widget , avoid repetition of widgets when you can manipulate the value of that widget.